src/components/add-note-form/AddNoteForm.jsx

```diff
@@ -36,7 +36,6 @@ export const AddNoteForm = ({ name }) => {
 
     if (isValid) {
       const currentHistoryState = notesLists[name];
-      // const currentAmountState = amountsState[activeTabState];
       const date = `${(new Date()).toLocaleDateString('ru-RU')} | ${(new Date()).toLocaleTimeString('ru-RU')}`;
       const newId = currentHistoryState.length;
```

Comment:
using length like id it's bad idea, use library https://www.npmjs.com/package/uuid